plugins/webapi/info/plugin.go

```diff
@@ -0,0 +1,92 @@
+package info
+
+import (
+	"net/http"
+
+	"github.com/iotaledger/goshimmer/plugins/autopeering/local"
+	"github.com/iotaledger/goshimmer/plugins/banner"
+	"github.com/iotaledger/goshimmer/plugins/webapi"
+	"github.com/iotaledger/hive.go/node"
+	"github.com/labstack/echo"
+	"github.com/mr-tron/base58"
+)
+
+var PLUGIN = node.NewPlugin("WebAPI info Endpoint", node.Enabled, configure)
+
+func configure(plugin *node.Plugin) {
+	webapi.Server.GET("info", getInfo)
+}
+
+// getInfo returns the info of the node
+// e.g.,
+// {
+// 	"version":"v0.2.0",
+// 	"identity":"7BxV1v3nFHefn4J88jeZebqnJRvSHt1jC7ME6tmKLhy7",
+// 	"publickey":"CjUsn86jpFHWnSCx3NhWfU4Lk16mDdy1Hr7ERSTv3xn9",
+// 	"enabledplugins":[
+// 		"Config",
+// 		"Autopeering",
+// 		"Analysis",
+// 		"WebAPI data Endpoint",
+// 		"WebAPI dRNG Endpoint",
+// 		"MessageLayer",
+// 		"CLI",
+// 		"Database",
+// 		"DRNG",
+// 		"WebAPI autopeering Endpoint",
+// 		"Metrics",
+// 		"PortCheck",
+// 		"SPA",
+// 		"WebAPI",
+// 		"WebAPI info Endpoint",
+// 		"WebAPI message Endpoint",
+// 		"Banner",
+// 		"Gossip",
+// 		"Graceful Shutdown",
+// 		"Logger"
+// 	],
+// 	"disabledlugins":[
+// 		"Graph",
+// 		"RemoteLog",
+// 		"Spammer",
+// 		"WebAPI Auth"
+// 	]
+// }
+func getInfo(c echo.Context) error {
+	enabledPlugins := []string{}
+	disabledPlugins := []string{}
```

Comment:
```suggestion
	var enabledPlugins []string
	var disabledPlugins []string
```
Not entirely sure but I think its more Go idiomatic to declare a nil slice instead of an empty literal declaration.